testutil/app/app.go

```diff
@@ -338,6 +344,15 @@ func NewRollapp(
 		),
 	)
 
+	app.DenommetadataKeeper = denommetadatakeeper.NewKeeper(
+		appCodec,
+		keys[denommetadatatypes.StoreKey],
+		app.BankKeeper,
+		nil,
+		app.GetSubspace(denommetadatatypes.ModuleName),
+	)
+	// set hook for denom metadata keeper later
```

Comment:
need to set empty MultiHooks otherwise code will panic on tx (as `k.hooks` is `nil`)

Reply:
pls address this issue